docs/Testing.md

```diff
@@ -0,0 +1,69 @@
+# Testing
+
+Moya has been created with testing at its heart. In this document, you will find all the customization points that will allow you to tailor your testing.
+
+## `sampleData`
+
+When creating your `TargetType` you are required to provide `sampleData` for your targets. All you need to do there is to provide `Data` that represents a sample response from every particular target.
+
+For example:
+
+```swift
+public var sampleData: Data {
+    switch self {
+    case .userRepositories(let name):
+        return "[{\"name\": \"Repo Name\"}]".data(using: String.Encoding.utf8)!
+    }
+}
+```
+
+This can be used later for tests or just for providing offline support while developing.
```

Comment:
I would move this line to the paragraph before "For example", as the last sentence.